mappings/net/minecraft/Bootstrapper.mapping

```diff
@@ -1,5 +1,5 @@
-CLASS net/minecraft/unmapped/C_ntskruzf net/minecraft/Bootstrap
-	FIELD f_jltdqkzo initialized Z
+CLASS net/minecraft/unmapped/C_ntskruzf net/minecraft/Bootstrapper
```

Comment:
i disagree with this rename, since it's an all-static class. additionally, `Bootstrap` has parity across yarn and moj

Reply:
I mainly did it to avoid `Bootstrap.bootstrap()`, bit ig that's fine.

Do you prefer the old "initialize" field+method names as well?

Reply:
i think the new ones are better!